schemas.md

```diff
@@ -43,6 +43,19 @@ The user's geolocation at that time.
 - `description`: Text description, such as messages sent by a GPS beacon.
 - `mixin:location`
 
+### activity.motion
```

Comment:
I'm not sure if it makes sense to namespace the attributes, as I did for `file` and `media`. I don't think it would be a problem, although media entries also have a `duration`.